samples/README.md

```diff
@@ -0,0 +1,147 @@
+[![Build Status](https://travis-ci.org/IBM/MAX-Image-Caption-Generator.svg?branch=master)](https://travis-ci.org/IBM/MAX-Image-Caption-Generator) [![Website Status](https://img.shields.io/website/http/max-image-caption-generator.max.us-south.containers.appdomain.cloud/swagger.json.svg?label=api+demo)](http://max-image-caption-generator.max.us-south.containers.appdomain.cloud/)
```

Comment:
It seems like the entire main readme is now here in samples? Not sure what is going on

Reply:
I'm not sure how that happened, thanks for catching